documentation/modules/overview/con-configuration-points-mirrormaker.adoc

```diff
@@ -36,10 +37,10 @@ Topic replication is defined using regular expression patterns to include or exc
 
 The following MirrorMaker 2.0 connectors and related internal topics help manage the transfer and synchronization of data between the clusters.
 
-MirrorSourceConnector:: A _MirrorSourceConnector_ creates remote topics from the source cluster.
+MirrorSourceConnector:: A _MirrorSourceConnector_ creates remote topics from the source cluster. It also replicates ACLs and is necessary for the `MirrorCheckpointConnector` to run.
```

Comment:
This seems to mirror the content from previous module. Same comment about ACL applies.

Reply:
Yes. This is in the _Overview_ guide. I'm going to split the replicated content and reuse it...